locust/test/test_dispatch.py

```diff
@@ -3291,6 +3291,241 @@ class User3(User):
         self.assertEqual(_user_count_on_worker(dispatched_users, worker_nodes[2].id), 3)
 
 
+class TestRampUpUsersFromZeroWithFixed(unittest.TestCase):
+    class Case:
+        def __init__(self, fixed_counts: Tuple[int], weights: Tuple[int], target_user_count: int):
+            self.fixed_counts = fixed_counts
+            self.weights = weights
+            self.target_user_count = target_user_count
+
+    def case_handler(self, cases: List[Case], expected: Dict[str, int], user_classes=List[User]):
+        self.assertEqual(len(cases), len(expected))
+
+        for case_num in range(len(cases)):
+            # Reset to defaul values
+            for user_class in user_classes:
+                user_class.weight, user_class.fixed_count = 1, 0
+
+            case = cases[case_num]
+            self.assertEqual(
+                len(case.fixed_counts) + len(case.weights),
+                len(user_classes),
+                msg="Invalid test case or user list.",
+            )
+
+            fixed_users_list = user_classes[: len(case.fixed_counts)]
```

Comment:
No need to use hungarian notation
```suggestion
            fixed_users = user_classes[: len(case.fixed_counts)]
```

Reply:
Fixed.